api/badges.ts

```
@@ -41,36 +48,54 @@ but it will change to either \`recaptime-dev\` or \`lorebooks-wiki\` in the futu
     },
   };
 
-  async handle(c: Context) {
+  override async handle(c: Context) {
     const apiReqData = await this.getValidatedData<typeof this.schema>();
-    const { org, style } = apiReqData.query;
+    const { org, style } = apiReqData?.query;
```

Biome

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)